deploy/clowdapp.yaml

```diff
@@ -16,10 +16,9 @@ objects:
       webServices:
         public:
           enabled: true
+          apiPath: patch
         private:
           enabled: false
-        metrics:
-          enabled: true
```

Comment:
What's the reason for removing `metrics`?

Reply:
setting 
```
        metrics:
          enabled: true
```
is wrong, there is no `enabled` stanza, we can't enable/disable it, it is there always by default, but let me double check the clowder api docs

Reply:
https://github.com/RedHatInsights/clowder/blob/master/config/crd/bases/cloud.redhat.com_clowdapps.yaml#L2721-L2726